test/diff/helper.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package diff
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+// ErrorIfWantNotGot Wrap cmp.Diff with a t.Helper(). It takes the testing.T
+// want,got, format message and cmp.option as an argument. Therefore, if diff was found
+// it will call t.Error of format and diff that wrapped by PrintWantGot func
+func ErrorIfWantNotGot(t *testing.T, want interface{}, got interface{}, format string, opts ...cmp.Option) {
+	// t.Helper() used to marks the calling function as a test helper function
```

Comment:
I don't think this comment is needed; a developer can quickly check the golang docs.

Reply:
okay, it's already removed 😄